acceptance/server_test.go

```diff
@@ -54,6 +54,7 @@ func AddHandlers(server *testserver.Server) {
 
 	server.Handle("GET /api/2.0/preview/scim/v2/Me", func(r *http.Request) (any, error) {
 		return iam.User{
+			Id:       "tester@databricks.com",
```

Comment:
Even though it's a string, for correctness it should still represent the number (int64)
https://docs.databricks.com/api/workspace/currentuser/me#id

Reply:
Yes, fixed here https://github.com/databricks/cli/pull/2266/commits/4b6211c46f06ba947a67da5c74ad25da4e3c19b5